mlx_graphs/nn/conv/gin_conv.py

```diff
@@ -0,0 +1,99 @@
+from typing import Optional
+
+import mlx.core as mx
+import mlx.nn as nn
+
+from mlx_graphs.nn.message_passing import MessagePassing
+
+
+class GINConv(MessagePassing):
+    r"""Graph Isomorphism Network convolution layer from `"How Powerful are
+    Graph Neural Networks?" <https://arxiv.org/abs/1810.00826>`_ paper.
+
+    .. math::
+        \mathbf{h}_i = \text{MLP} \left( (1 + \epsilon) \cdot
+        \mathbf{x}_i + \sum_{j \in \mathcal{N}(i)} \mathbf{x}_j \right)
+
+    where :math:`\mathbf{x}_i` and  :math:`\mathbf{h}_i` represent the input features
+    and output embeddings of node :math:`i`, respectively. :math:`\text{MLP}` denotes
+    a custom neural network provided by the user and :math:`\epsilon` is an epsilon
+    value either fixed or learned.
+
+    Args:
+        mlp: Callable :class:`mlx.core.nn.Module` applied on the final node embeddings
+        eps (optional): Initial value of the :math:`\epsilon` term. Default: ``0``
```

Comment:
Please remove all the `(optional)` from the docstrings. Types are inferred from the signature and if a value is provided, like `(optional)`, the actual type is overwritten in the docs.

Reply:
You're right